src/i18n/es.js

```diff
@@ -212,6 +212,8 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
     delivery_rate: `Tasa de entrega`,
     emails_delivered: `Emails Entregados:`,
     hard_and_soft: `Rebotes Hard y Soft`,
+    header_description_shipped: `Este es un Reporte de tu Campaña enviada.`,
```

Comment:
```suggestion
    header_description_shipped: `Este es un Reporte de tu Campaña Enviada.`,
```
![image](https://user-images.githubusercontent.com/2439363/79792191-dd1dcb00-8324-11ea-9e0b-377233b44913.png)